android/assets/i18n/messages.properties

```diff
@@ -38,35 +38,35 @@ game-select.high-score=High Score
 
 unimplemented-game.title=More Games Coming Soon!
 unimplemented-game.description=Know of a good game to work on next?\nVote for it on GitHub.
-unimplemented-game.next-game-dev=When we get time to add another game,\nthose with many thumbs up will most likely come next.
+unimplemented-game.next-game-dev=When we get time to add another game,\nthose with many upvotes will most likely come next.
 unimplemented-game.coming-soon=More\ncoming soon!
 unimplemented-game.vote=Vote
 
-multiplayer.lobby.splash.play-others=Play other retro fans\nover the internet
-multiplayer.lobby.splash.play-friends=Play friends on your WiFi network
+multiplayer.lobby.splash.play-others=Play other retro fans\nover the Internet
+multiplayer.lobby.splash.play-friends=Play with friends on your Wi-Fi network
 multiplayer.lobby.splash.btn.play-online=Play online
 multiplayer.lobby.splash.btn.start-local-server=Start local server
 multiplayer.lobby.splash.btn.join-local-server=Join local server
 
 multiplayer-lobby.title=Multiplayer
 
-multiplayer.local.starting-server=Starting server
-multiplayer.local.waiting-for-others=Waiting for other players to join
+multiplayer.local.starting-server=Starting server…
```

Comment:
Although the `…` is preferable (and indeed Weblate will recommend this over three individual dots `...`), the bitmap font used by the game doesn't have a glyph for this, hence sticking with `...` for now.